File: Clojure/0001multiples_of_3_or_5.clj
;;
;; Clojure 1.12.0.1530
;; 0001multiples_of_3_or_5.clj
;; 
;; Copyright 2025 @GitHub JoergEm
;;
;; This program is WITHOUT ANY WARRANTY; without even the implied
;; warranty of MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.
;;

(defn summe [nummer]
  (reduce + (range nummer 1000 nummer) ) )
(defn multiples_of_3_or_5 []
  (- (+ (summe 3) (summe 5) ) (summe 15) ) )
(println (multiples_of_3_or_5) )

File: C/0001multiples_of_3_or_5.c
/* 
 * gcc 12.2.0
 * 0001multiples_of_3_or_5.c
 * 
 * Copyright 2025 @GitHub JoergEm
 * 
 * This program is WITHOUT ANY WARRANTY; without even the implied
 * warranty of MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.
 * 
 * 
 */


#include <stdio.h>

int main() {
    int summe = 0;

    for (int nummer = 1; nummer < 1000; nummer++) {
        if (nummer % 3 == 0 || nummer % 5 == 0) {
            summe += nummer;
        }
    }

    printf("%d\n", summe);
    return 0;
}